.github/workflows/build-extension.yml

```diff
@@ -73,7 +73,20 @@ jobs:
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
 
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: yarn
```

Comment:
Why is it required to checkout and run yarn?

If needed, would be better to use provision script, that has caching in place

Reply:
If I don't have this, then it fails. I'll try the provision script. 👍  It was part of the Chromatic docs example I followed.